docs/dev/konnector.md

```diff
@@ -42,24 +41,11 @@ Then, write your code into `konnectors.js` and build the application, running `y
 
 ### Collector structure
 
-Basically, a collector is just an object with some required fields:
+Basically, a connector is just an object passed to baseKonnector.createNew:
 
- - `name`: the displayed name of the collector;
```

Comment:
These attributes are now defined in the manifest.konnector file. We still need to define name and models just due to compatibility code with v2 but this will be removed